src/components/topic/topic.jsx

```diff
@@ -76,7 +76,7 @@ export default class Topic extends React.Component {
 
                 <section className={ `${block}__vote` }>
                     <div className={ `${block}__title ${block}__title--vote` }>
-                        Place Your Vote
+                        {topic.locked ? 'Voting not allowed' : 'Place your vote'}
```

Comment:
Im sorry, grammar is off:
```suggestion
                        {topic.locked ? 'Voting is not allowed' : 'Place your vote'}
```

Reply:
Fixed.